### ISSUE TITLE
`merge` is not documented in the osmium-tool manual

### ISSUE THREAD

**Issue by stouch** [osmcode/osmcode.github.io#13]
https://osmcode.org/osmium-tool/manual.html does not mention `merge` command, which leads people to ask it here for example : https://gis.stackexchange.com/questions/242704/merging-osm-pbf-files

**Comment by joto** [comment]
Yep, that manual incomplete. Full documentation of all commands is here: https://docs.osmcode.org/osmium/latest/
